content/applications/finance/expenses/reimburse.rst

```diff
@@ -0,0 +1,134 @@
+
+.. _expenses/reimburse-bulk:
+
+Reimburse in bulk
+=================
+
+To reimburse multiple expense reports at once, navigate to :menuselection:`Expenses app --> Expense
+Reports --> Reports To Pay` to view all expense reports in a list view. Next, adjust the
+:guilabel:`STATUS` filters on the far-left side to only present expense reports with a status of
+:guilabel:`POSTED`.
+
+.. tip::
+   Adjusting the :guilabel:`STATUS` filters to only show :guilabel:`POSTED` expense reports is not
+   necessary, but removes the step of selecting each individual report in the list.
+
+Tick the checkbox next to :guilabel:`Employee` to select all the reports in the list. Once ticked,
+the number of selected expense reports appears on a :guilabel:`(#) Selected` button in the
+top-center of the screen. Additionally, a :guilabel:`Register Payment` smart button also appears, on
+the left-hand side.
+
+.. image:: reimburse/multiple-reports.png
+   :align: center
+   :alt: Expense reports filtered by the status Posted, making the Register Payment button visible.
+
+Click the :guilabel:`Register Payment` button, and a :guilabel:`Register Payment` pop-up window
+appears. Enter the following information in the pop-up window:
+
+- :guilabel:`Journal`: Select the accounting journal to post the payment to, using the drop-down
+  menu.
+- :guilabel:`Payment Method`: Select how the payment is made using the drop-down menu. The default
+  options are :guilabel:`Manual`, :guilabel:`Checks`, and :guilabel:`NACHA`.
+- :guilabel:`Group Payments`: If multiple expense reports are selected for the same employee, this
+  option appears. Tick the checkbox to have only *one* payment made per employee, rather than
+  issuing multiple payments to the same employee.
+- :guilabel:`Payment Date`: Enter the date the payments are issued. The current date populates this
+  field by default.
+
+.. image:: reimburse/register.png
+   :align: center
+   :alt: The Register Payment pop-up window filled out.
+
+When the form is completed, click the :guilabel:`Create Payments` button to register the payments,
+and reimburse the employees.
+
+.. _expenses/reimburse-payslip:
+
+Report in next payslip
+======================
+
+If the option to reimburse employees in their payslips is activated in the :guilabel:`Expenses`
+section, payments can be added to their next payslip, instead of issued manually.
+
+.. important::
+   Reimbursing expenses on payslips can **only** be done individually, on an expense report with a
+   status of :guilabel:`Approved`. Once an expense report has a status of :guilabel:`Posted`, the
+   option to reimburse in the following payslip does **not** appear.
+
+Navigate to :menuselection:`Expenses app --> Expense Reports --> Reports To Pay`, and click on the
+individual expense report being reimbursed on the following paycheck. Click the :guilabel:`Report in
+Next Payslip` smart button, and the expenses are added to the next payslip issued for that employee.
```

Comment:
```suggestion
Next Payslip` button, and the expenses are added to the next payslip issued for that employee.
```

Reply:
Do you think it would be beneficial to add that a comment about it being added to their next payslip appears in teh chatter?